pnpm-lock.yaml

```diff
@@ -18995,7 +18997,7 @@ packages:
     dependencies:
       '@inquirer/type': 1.2.1
       '@types/mute-stream': 0.0.4
-      '@types/node': 20.12.6
+      '@types/node': 20.12.10
```

Comment:
Probably revert by hand these changes to this file that are not about adding the telemetry-utils dependency.